### DISCUSSION TITLE
Disabling Charts for User Groups and Applications, with Selective Chart Enable for User Seaction in Netdata Configuration

### DISCUSSION THREAD

**Discussion by ghost** [netdata/netdata#17349] [Q&A]
**Description:**

Hello everyone,

I'm currently facing a challenge with selectively disabling specific charts within the Users section of my Netdata configuration (`netdata.conf`). Here's a breakdown of my setup and what I've tried:

In my `netdata.conf`, I have enabled the Apps plugin:

```conf
[plugins]
    apps = yes
```

This allows me to view charts for User Groups, Users, and Applications.
![Screenshot 2024-04-09 152348](https://github.com/netdata/netdata/assets/87641643/12015eb0-7275-4b7e-8ba7-c08cd451c286)

However, I only need to display follwing charts from the Users section to show this in my Grafana graphs using InfluxDB:

1. Users Disk Writes (`users.pwrites`)
2. Users Disk Logical Reads (`users.lreads`)
3. Users I/O Logical Writes (`users.lwrites`)
4. Users Real Memory (w/o shared) (`users.mem`)
![Untitled](https://github.com/netdata/netdata/assets/87641643/d48b7884-965d-4772-a06a-67882c1ae999)

Initially, I attempted to disable all charts from the Users, User Groups, and Applications sections using the following configuration:

```conf
[plugin:apps]
    command options = no-users no-groups no-apps
```

However, this approach disabled all charts from the specified sections, which is not what I intended.

I then tried to disable the User Groups and Applications charts separately, but it does not work.

```conf
# Disable the "User Groups" Charts
[plugin:apps]
    command options = no-groups no-apps
```
Unfortunately, these configurations didn't work as expected.

Subsequently, I attempted to disable individual charts within the Users and Applications sections using the `enabled = no` option, please check below its works as expected but this method lacked the desired granularity and flexibility.

```
# Per chart configuration
# Disabled Applications chart
[apps.cpu]
	enabled = no

[apps.cpu_user]
	enabled = no

[apps.cpu_system]
	enabled = no

[apps.preads]
	enabled = no

[apps.pwrites]
	enabled = no

[apps.lreads]
	enabled = no

[apps.lwrites]
	enabled = no

[apps.files]
	enabled = no

[apps.mem]
	enabled = no

[apps.vmem]
	enabled = no

[apps.minor_faults]
	enabled = no

[apps.major_faults]
	enabled = no

[apps.threads]
	enabled = no

[apps.processes]
	enabled = no

[apps.uptime]
	enabled = no

[apps.uptime_min]
	enabled = no

[apps.uptime_avg]
	enabled = no

[apps.uptime_max]
	enabled = no

[apps.pipes]
	enabled = no

[apps.swap]
	enabled = no

[apps.sockets]
	enabled = no
```
```
# Disabled Users chart
[users.cpu]
	enabled = no

[users.vmem]
	enabled = no

[users.threads]
	enabled = no

[users.processes]
	enabled = no

[users.uptime]
	enabled = no

[users.uptime_min]
	enabled = no

[users.uptime_max]
	enabled = no

[users.uptime_avg]
	enabled = no

[users.cpu_user]
	enabled = no

[users.cpu_system]
	enabled = no

[users.swap]
	enabled = no

[users.major_faults]
	enabled = no

[users.minor_faults]
	enabled = no

[users.lreads]
	enabled = no

[users.lwrites]
	enabled = no

[users.files]
	enabled = no

[users.sockets]
	enabled = no

[users.sockets]
	enabled = no

[users.pipes]
	enabled = no
```

I also tried to disable all charts of users, groups, and applications at once.
Please check below if there is any regex condition available to disable all charts at once.
```
[users.*]
	enabled = no

[apps.*]
	enabled = no

[groups.*]
	enabled = no
```

I'm now seeking alternative solutions or insights from the community on how to achieve my objective of selectively disabling specific charts within the Users section while keeping the Apps plugin enabled.

Any help or suggestions would be greatly appreciated.

Thank you in advance for your assistance!

**Comment by ilyam8** [comment]
Hi, @fti-sfuke. I do not think that's possible.

> I only need to display follwing charts from the Users section to show this in my Grafana graphs using InfluxDB

You can [filter out charts when exporting](https://learn.netdata.cloud/docs/exporting-metrics/exporting-reference#chart-filtering).
